### PR TITLE
Completing translation of ch06-02-match.md

### PR DESCRIPTION
**Pull request by dbucci24**
completing Spanish translation of ch06-02-match.md from line 193 to 299

-599 words

### CODE REVIEW

File: second-edition/src/ch06-02-match.md [hunk 1]
```diff
@@ -190,55 +190,57 @@ let five = Some(5);
...
-`plus_one(five)`, the variable `x` in the body of `plus_one` will have the
-value `Some(5)`. We then compare that against each match arm.
 
 ```rust,ignore
 None => None,
 ```
 
-The `Some(5)` value doesn’t match the pattern `None`, so we continue to the
-next arm.
+El valor `Some(5)` no coincie con el patrón `None`, entonces continuamos al siguiente
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: coincide

File: second-edition/src/ch06-02-match.md [hunk 2]
```diff
@@ -190,55 +190,57 @@ let five = Some(5);
...
 
 ```rust,ignore
 Some(i) => Some(i + 1),
 ```
 
-Does `Some(5)` match `Some(i)`? Well yes it does! We have the same variant.
-The `i` binds to the value contained in `Some`, so `i` takes the value `5`. The
-code in the match arm is then executed, so we add one to the value of `i` and
-create a new `Some` value with our total `6` inside.
+Coincide `Some(5)` con `Some(i)`? Si lo hace! Tenemos la misma variante.
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: Please review spanish punctuation signs must both open and close

File: second-edition/src/ch06-02-match.md [hunk 3]
```diff
@@ -190,55 +190,57 @@ let five = Some(5);
...
+La `i` se une al valor contenido en `Some`, entonces `i` toma el valor `5`. El
+código en el brazo de coincidencia es entonces ejecutado, asique agregamos uno
+al valor de `i` y creamos un nuevo valor `Some` con nuestro total `6` adentro.
 
-#### Matching `None`
+#### Emparejando `None`
+
+Ahora consideremos la segunda llamda de `plus_one` en el Listado 6-5 donde `x` es
+`None`. Ingreamos el valor `match` y lo comparamos al primer brazo.
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: Ingresamos

File: second-edition/src/ch06-02-match.md [hunk 4]
```diff
@@ -190,55 +190,57 @@ let five = Some(5);
...
-It matches! There’s no value to add to, so the program stops and returns the
-`None` value on the right side of `=>`. Because the first arm matched, no other
-arms are compared.
+Coincide! No hay valor para agregar, entonces el programa se detiene y devuelve el valor
+`None` en el lado derecho de `=>`. Debido a que el primer brazo coincide, no se 
+comparan otros brazos.
+
+Combinar `match` y enumaraciones en útil en muchas situaciones. Verás mucho este patrón
+en el código Rust: `match` contra una enumeración, junta una variable con los
+datos adentro, y entonces ejecuta código basado en él. Es un poco díficil ap principio, pero
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: al

File: second-edition/src/ch06-02-match.md [hunk 5]
```diff
@@ -190,55 +190,57 @@ let five = Some(5);
...
+una vez que lo usas, querrás tenerlo en todos los lenguajes. Es
+consistentemente un usuario favorito.
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: *una de las funciones favoritas de los usuarios.

File: second-edition/src/ch06-02-match.md [hunk 6]
```diff
@@ -248,8 +250,8 @@ fn plus_one(x: Option<i32>) -> Option<i32> {
 }
 ```
 
-We didn’t handle the `None` case, so this code will cause a bug. Luckily, it’s
-a bug Rust knows how to catch. If we try to compile this code, we’ll get this
+No manejamos el caso `None`, entonces este código causará un bug. Por suerte, es un
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: *es un bug

File: second-edition/src/ch06-02-match.md [hunk 7]
```diff
@@ -260,20 +262,21 @@ error[E0004]: non-exhaustive patterns: `None` not covered
...
-Rust knows that we didn’t cover every possible case and even knows which
-pattern we forgot! Matches in Rust are *exhaustive*: we must exhaust every last
-possibility in order for the code to be valid. Especially in the case of
-`Option<T>`, when Rust prevents us from forgetting to explicitly handle the
-`None` case, it protects us from assuming that we have a value when we might
-have null, thus making the billion dollar mistake discussed earlier.
 
-### The `_` Placeholder
+Rust sabe que no cubrimos cada caso posible every possible e incluso sabe que patrón
+olvidamos! Las coincidencias en Rust son *exhaustive*: debemos usar hasta la última
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: Please translate exhauste/exhaustive by agotar/agotadores

File: second-edition/src/ch06-02-match.md [hunk 8]
```diff
@@ -286,11 +289,11 @@ match some_u8_value {
...
+El patrón `_` concidirá cualquier valor. Poniéndolo después de nuestros otros brazos, el
+`_` coincidierá con todos los casos posibles que no fueron especificados antes. El `()`
+es solo la unidad de valor, por lo que nada pasará en el caso  `_`. Como resultado, podemos 
+decir que no queremos hacer nada para todos los valores posibles que no listamos
+antes del marcador de posición `_` .
 
-However, the `match` expression can be a bit wordy in a situation in which we
-only care about *one* of the cases. For this situation, Rust provides `if let`.
+Sin embargo, la expresión `match` puede ser un poco tediosa en una situación en que solo
+nos importa *one* de los casos. Para esta situacuión, Rust provee `if let`.
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: *uno\8

**Comment by dbucci24** [comment]
fixed!

**Comment by thecodix** [comment]
This PR still needs some work, specially on the functions (variable names and stuff), but I expect you to compromise and finish it 😣

**Comment by dbucci24** [comment]
I will work on it :)
